### ISSUE TITLE
The repair/reforge material of MI Quantum Armor should be a Singularity from MI.

### ISSUE THREAD

**Issue by James103** [TeamAOF/All-of-Fabric-6#342]
Currently, Quantum Armor from Modern Industrialization can be reforged using flint, which I feel is way too cheap for the armor's benefits (full set bonus: complete immunity to all damage).

To fix this, I suggest making it so that reforging MI Quantum Armor should cost a Singularity from MI, to incentivize mass production of them. This can be done by adding 5 files to the `data/tiered/reforge_item` folder of a data pack, each with the following content.
```json
{
    "product": {
        "id": "modern_industrialization:quantum_X"
    },
    "cover": true,
    "base": [
        {
            "id": "modern_industrialization:quantum_singularity"
        }
    ]
}
```
(replacing X with `helmet`, `chestplate`, `leggings`, `boots` and `sword` and saving the result after each replacement to a separate file)

**Comment by Trent2087** [comment]
yup, saw the other issue about reforging that showed how to make datapacks to fix some other armors. Note that the only reason MI quantum is reforgable this way is because its an upgrade from netherite, so the nbt data carries over. if you cheat in MI Quantum it is not reforgable. as the repair ingredient is null.
